spec/features/visitors/sign_up_spec.rb

```diff
@@ -255,19 +242,6 @@
     expect(current_path).to eq user_confirmation_path
   end
 
-  context 'confirmation instructions sent to existing user', email: true do
-    xit 'does not send an email to the existing user' do
```

Comment:
this was set to pending here https://github.com/18F/identity-idp/pull/564/files#r83312773

I think it makes sense to remove until we add in this feature. I love a pending-spec-free test suite!